bibliopixel/layout/circle.py

```diff
@@ -1,7 +1,7 @@
 from . layout import Layout
 from . geometry.circle import (
     gen_circle, calc_ring_steps, calc_ring_pixel_count,
-    pixel_positions_from_rings)
+    gen_circle_positions)
```

Comment:
I cleaned up these names because clearly they were inconsistent and causing confusion before.